Added description about default.xml (#4576)
magento/devdocs: guides/v2.1/frontend-dev-guide/layouts/xml-instructions.md

To change the page wireframe, modify the [page layout]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-page) files; all other customizations are performed in the [page configuration]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-conf) or [generic layout]({{page.baseurl}}/frontend-dev-guide/layouts/layout-types.html#layout-types-gen) files. 

> ## Layout handle

Reviewer: ```suggestion
## Manage layouts
```

> In order to add a layout changes that will be available on every page, a module should use `deafult.xml` layout.

Reviewer: ```suggestion
To make layout changes available on every page, modify the `default.xml` file.
```

> For example, layout changes that is added to `app/code/Vendor/Module/view/frontend/layout/default.xml` will be loaded on all pages.

Reviewer: ```suggestion
For example, layout changes added to `app/code/Vendor/Module/view/frontend/layout/default.xml` are loaded on all pages.
```

> To add layout changes to a specific page a module should use layout handle that corresponds the page's path. 

Reviewer: ```suggestion
To add layout changes to a specific page, use a layout file that corresponds to the page's path. 
```

> To add layout changes to a specific page a module should use layout handle that corresponds the page's path. 

Reviewer: What is a "layout handle"?

> For example, the changes that are added to `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` will be reflected on the product details page.

Reviewer: ```suggestion
For example, changes to the `app/code/Vendor/Module/view/frontend/layout/catalog_product_view.xml` file are loaded on the product details page.
```
